packages/functional-tests/tests/misc/keyStretchingV2.spec.ts

```diff
@@ -131,7 +137,7 @@ test.describe('key-stretching-v2', () => {
         await page.goto(
           `${target.contentServerUrl}/?showReactApp=true&forceExperiment=generalizedReactApp&forceExperimentGroup=react&${stretch}`
         );
-        await signupReact.fillOutEmailFirst(email);
+        await signupReact.fillOutEmailForm({ email: email, submit: true });
```

Comment:
Tiny syntactical sugar nit, `email: email` can just be `email`.

One improvement, could we set `submit` to `true` by default in this method so we only need to pass `submit: false` when we don't want it to submit?

Reply:
Oh syntactical sugar is nice!!!

I intentionally took away the default on submit, but am re-thinking now because your comment makes me think perhaps the submit action is easily assumed. Also after some inspection, I think we don't have a use case where it needs to be false...